docs/elements.md

```diff
@@ -1902,6 +1903,46 @@ Note that only one of the attributes `source-file-name`, `submitted-file-name` o
 
 - [`pl-file-editor` to provide an in-browser code environment](#pl-file-editor-element)
 
+### `pl-card` element
+
+Displays question content within a card-styled component. Optionally displays a header, footer, and/or image via tag attributes.
+
+#### Sample element
+
+```html
+<pl-card
+  header="Header"
+  title="Title"
+  width="50%"
+  img-bottom-src="https://via.placeholder.com/720x480"
+>
+  <pl-question-panel> This card is 50% width and has a bottom image. </pl-question-panel>
+</pl-card>
+```
+
+#### Customizations
+
+| Attribute        | Type   | Default | Description                             |
+| ---------------- | ------ | ------- | --------------------------------------- |
+| `header`         | string | -       | Contents of the card header             |
+| `title`          | string | -       | Contents of the card title              |
+| `subtitle`       | string | -       | Contents of the card subtitle           |
+| `contents`       | string | -       | Raw contents of the card body           |
+| `footer`         | string | -       | Contents of the card footer             |
+| `img-top-src`    | string | -       | Source URL for the top image            |
+| `img-top-alt`    | string | -       | Alternative text for the top image      |
+| `img-bottom-src` | string | -       | Source URL for the bottom image         |
+| `img-bottom-alt` | string | -       | Alternative text for the bottom image   |
+| `width`          | string | auto    | Width of the card (25%, 50%, 75%, auto) |
```

Comment:
This is how we express enum-ish values in other elements (will require reformatting this table with Prettier):

```suggestion
| `width`          |"25%", "50%", "75%", or "auto" | auto    | Width of the card |
```